lib/tasks/export_portage_1.rake

```diff
@@ -0,0 +1,51 @@
+# To maintain the original hierarchy, all items below needs to keep the original id
+
+require 'faker'
+
+namespace :export do
+  desc "Export org, guidance_group and guidance from 3.0.2 database" 
+  task :export_portage_1 => :environment do
+    excluded_keys = ['created_at','updated_at'] 
+    Org.all.each do |org|
+      if org.id == 8 # super admin's org: Portage network/Alliance in production database
```

Comment:
What do you think about fetching the first super administrator's org and working directly with it instead of depending on the current state of the production database?